pkg/dataset/run.go

```diff
@@ -83,26 +78,37 @@ func Run(pncBaseUrl, indyBaseUrl, buildId string) {
 	}
 
 	//Download group-build.json or build.json
+	var buildJsonFileLoc string
 	if isGroupBuild {
-		fileLoc = path.Join(dirLoc, "group-build.json")
-		if !common.FileOrDirExists(fileLoc) {
-			success := common.DownloadFile(groupBuildURL, fileLoc)
+		buildJsonFileLoc = path.Join(dirLoc, "group-build.json")
+		if !common.FileOrDirExists(buildJsonFileLoc) {
+			success := common.DownloadFile(groupBuildURL, buildJsonFileLoc)
 			if !success {
 				fmt.Println("Download group-build.json failed.")
 				return
 			}
-			formatJsonFile(fileLoc)
 		}
 	} else {
 		buildURL := pncBaseUrl + "/pnc-rest/v2/builds/" + buildId
-		fileLoc = path.Join(dirLoc, "build.json")
-		if !common.FileOrDirExists(fileLoc) {
-			success := common.DownloadFile(buildURL, fileLoc)
+		buildJsonFileLoc = path.Join(dirLoc, "build.json")
+		if !common.FileOrDirExists(buildJsonFileLoc) {
+			success := common.DownloadFile(buildURL, buildJsonFileLoc)
 			if !success {
 				fmt.Println("Download build.json failed.")
 				return
 			}
-			formatJsonFile(fileLoc)
+		}
+	}
```

Comment:
Maybe we can extract a function here to avoid these two duplicated if...else... code block?

Reply:
Ok.